test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -998,9 +1003,43 @@ ZoneId=$FileType
             Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy ByPass
             Get-ExecutionPolicy -Scope CurrentUser | Should Be "ByPass"
         }
+    }
+
+    Describe "Validate Set-ExecutionPolicy -Scope (Admin)" -Tags @('CI', 'RequireAdminOnWindows') {
+
+        BeforeAll {
+            if ($IsNotSkipped)
+            {
+                $originalPolicies = Get-ExecutionPolicy -list
+            }
+        }
+
+        AfterAll {
+            if ($IsNotSkipped)
+            {
+                RestoreExecutionPolicy $originalPolicies
+            }
+        }
+
+        It '-Scope LocalMachine is Settable, but overridden' {
+            # setup
+            Set-ExecutionPolicy -Scope Process -ExecutionPolicy Undefined
+            Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy Restricted
+            $errorId = $null
+            try
+            {
+                Set-ExecutionPolicy -Scope LocalMachine -ExecutionPolicy ByPass
+            }
+            catch [System.Security.SecurityException] {
+                $errorId = $_.FullyQualifiedErrorId
+            }
+
+            $errorId | Should Be 'ExecutionPolicyOverride,Microsoft.PowerShell.Commands.SetExecutionPolicyCommand'
+            Get-ExecutionPolicy -Scope LocalMachine | Should Be "ByPass"
```

Comment:
>Get-ExecutionPolicy -Scope LocalMachine | Should Be "ByPass" [](start = 12, length = 60)

I don't quite understand this:
attempt to set the localmachine policy, which is supposed to fail
then check the localmachine policy to see if it is what you just tried to set it to?
Shouldn't you get the localmachine policy in the BeforeAll, store it, then check against that value (rather than assume that ByPass will be correct)